hyp3_testing/helpers.py

```diff
@@ -107,3 +108,20 @@ def job_tifs(job_id, api, directory, keep=False):
             for ff in product_dir.rglob('*'):
                 ff.unlink()
             product_dir.rmdir()
+
+
+@contextmanager
+def archive_tifs(product_name, product_urls, directory, keep=False):
+    write_credentials_to_netrc_file(os.environ.get('EARTHDATA_LOGIN_USER'), os.environ.get('EARTHDATA_LOGIN_PASSWORD'))
+    product_dir = directory / product_name
+    if not product_dir.exists():
```

Comment:
What's the point of allowing for the possibility of `product_dir` already existing? (I see it's the same in the `job_tifs` function above.) Is it for running the golden tests locally? Not necessarily a problem, just curious.

Reply:
Yeah it's for running locally. If the context manager sees that the folder is already there (i.e., the data is already downloaded) it won't download it again.